addons/sys_intercom/fnc_intercomListRxTxActions.sqf

```diff
@@ -0,0 +1,50 @@
+/*
+ * Author: ACRE2Team
+ * Generates a list of modes for the intercom network of a vehicle: receive, transmit or receive and transmit.
+ *
+ * Arguments:
+ * 0: Vehicle <OBJECT>
+ * 1: Unit <OBJECT>
+ * 2: Arguments <ARRAY>
+ *   0: Intercon Network <NUMBER>
+ *   1: Functionality <NUMBER>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [cursorTarget] call acre_sys_intercom_fnc_intercomListRxTxActions
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_target", "_player", "_args"];
+_args params ["_intercomNetwork", "_functionality"];
+
+private _actions = [];
+switch (_functionality) do {
+    case INTERCOM_DISCONNECTED: {
+        WARNING_1("Entered no monitor in ace interaction menu for radio %1", _radio);
+    };
+    case INTERCOM_RX_ONLY: {
+        private _action = ["acre_trans_only", localize LSTRING(transOnly), "", {[_target, _player, _this select 2, INTERCOM_STATIONSTATUS_CONNECTION, INTERCOM_TX_ONLY] call FUNC(setStationConfiguration)}, {true}, {}, _intercomNetwork] call ace_interact_menu_fnc_createAction;
```

Comment:
Let's do `acre_intercom_xyz` to make sure we never run into conflicts with this.